docs/iidm/model/index.md

```diff
@@ -3,8 +3,12 @@ title: Internal data model
 layout: default
 ---
 
+Powsybl's internal data model describes electrical networks via [a network core model](#network-core-mode) which models the network equipments
```

Comment:
I think there is a small typo in the anchor: #network-core-mode**l**

I think we should also reduce the usage of **model** in this sentence (3 times)